aws-lambda-java-events/src/main/java/com/amazonaws/services/lambda/runtime/events/LambdaDestinationEvent.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with
+ * the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package com.amazonaws.services.lambda.runtime.events;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.joda.time.DateTime;
+
+import java.io.Serializable;
+import java.util.Map;
+
+/**
+ * Class to represent an invocation record for a Lambda event.
+ *
+ * @see <a href="https://docs.aws.amazon.com/lambda/latest/dg/invocation-async.html>Asynchronous invocation</a>
+ *
+ * @author msailes <msailes@amazon.co.uk>
+ */
+@NoArgsConstructor
+@Data
+public class LambdaDestinationEvent implements Serializable, Cloneable {
+
+    private String version;
+    private DateTime timestamp;
+    private RequestContext requestContext;
+    private Map<String, Object> requestPayload;
+    private ResponseContext responseContext;
+    private ResponsePayload responsePayload;
+
+    @NoArgsConstructor
+    @Data
+    public static class RequestContext implements Serializable, Cloneable {
+        private String requestId;
+        private String functionArn;
+        private String condition;
+        private int approximateInvokeCount;
+    }
+
+    @NoArgsConstructor
+    @Data
+    public static class ResponseContext implements Serializable, Cloneable {
+        private String statusCode;
```

Comment:
Looking at the AWS doc for the LambdaDestinationEvent I think statusCode should be an int type.

Reply:
Fixed.